docs/apache-airflow/concepts/timetable.rst

```diff
@@ -55,15 +55,17 @@ Built In Timetables
 Airflow comes with several common timetables built in to cover the most common use cases. Additional timetables
 may be available in plugins.
 
+.. _CronTriggerTimetable
+
 CronTriggerTimetable
 ^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Another timetable which accepts a cron expression. It behaves more compliant with standard cron than ``CronDataIntervalTimetable``.
+A cron compliant timetable.
```

Comment:
What does “cron compliant” mean? Cron is not a standard to be compliant to. Even if it is, this is still missing a lot of explaination what makes something cron compliant.

I would probably suggest ditch this compliance idea altogether, and simply explain how this timetable actually behaves, and compare it to the cron daemon’s behaviour, and potentially differences.

Reply:
Yeah. Cron-compliant is not good name. 

I think "working as cron" as an idea is pretty sound as people have good mental model for how cron works in general. And we know traditionally Airflow is definitely not "working like cron" despite having "cron-like schedule definition".

Something `cron-like triggering behaviour` (but that's also not a very cool name) is more something I'd use and @uranusjr is right that we should describe it in a bit more detail, especially that there are many cron variants bahving slightly difference.

Reply:
Stopped using "cron compliant" and created a separated section to tell the differences between the two timetables. Hope it makes more sense for the readers: https://github.com/apache/airflow/pull/23662/commits/92ab13b3a73aa746ce9afd25fc63c29feb6acef8